package.json

```diff
@@ -37,17 +36,12 @@
     "express-rate-limit": "^7.1.5",
     "fathom-client": "^3.6.0",
     "feed": "^4.2.2",
-    "follow-redirects": "^1.15.6",
     "front-matter": "^4.0.2",
-    "gray-matter": "^4.0.3",
-    "gunzip-maybe": "^1.4.2",
     "isbot": "^4",
-    "iso-639-1": "^3.1.0",
     "lru-cache": "^10.1.0",
     "luxon": "^3.4.4",
     "morgan": "^1.10.0",
     "octokit": "^3.1.2",
```

Comment:
If this isn't gone I don't think we're totally decoupled from github?

Looks like `lib/resources.server` and `lib/github.server` both use it, and some `routes/_extras.resources` files.  Those should also all be able to be deleted and then we can remove the dep?

Reply:
Correct, I meant to call this out so thanks for commenting on it

I'm a bit hesitant to delete them just in case someone is linking. How do you feel about me adding them to the v2 docs as part of the pre-build and setting up some redirects? Or am I being ridiculous and we should just bite the bullet?

Reply:
Agreed offline this is safe. Will follow up and move the resources to prerendering on v2 docs for posterity